templates/app-autoscaler.yml

```diff
@@ -159,7 +159,7 @@ instance_groups:
     - name: postgres
       release: postgres
       properties:
-        databases: &database
+        databases: &database   
```

Comment:
👀 

```suggestion
        databases: &database
```

Reply:
Not sure why you _resolved_ this comment but I still see the spaces which were introduced but not required